packages/components/data-table-manager/src/data-table-settings/data-table-settings.tsx

```diff
@@ -223,23 +214,34 @@ const DataTableSettings = (props: TDataTableSettingsProps) => {
       <Spacings.Inline justifyContent="space-between" alignItems="center">
         <TopBarContainer>{props.topBar}</TopBarContainer>
         {dropdownOptions.length > 0 && (
-          <SelectContainer>
+          <>
             <AccessibleHidden>
               <label htmlFor="table-settings-dropdown">
                 Open table manager dropdown
               </label>
             </AccessibleHidden>
-            <SelectInput
-              id="table-settings-dropdown"
-              // the dropdown always shows the placeholder as selecting an option
-              // will open the corresponding panel (column manager or display settings)
-              value=""
-              placeholder={intl.formatMessage(messages.placeholder)}
-              onChange={handleDropdownChange}
-              options={dropdownOptions}
-              iconLeft={<TableIcon color="neutral60" />}
-            />
-          </SelectContainer>
+            <Tooltip placement="left" title="Table settings">
+              <DropdownMenu
+                triggerElement={
+                  <IconButton icon={<ColumnsIcon />} label="list" />
+                }
+                menuHorizontalConstraint={4}
```

Comment:
`menuHorizontalContraint` with value of `3` would wrap the text in the dropdown so I upped it to `4` as that was the closest I could get to the design and ensuring no wrapping occurred.

Reply:
I think we need to tag @FilPob here to confirm to approve of this decision.

Reply:
I think that's a good option, but bear in mind this adjustment works for english translation but might not work for other languages.
For instance, if we use portuguese, the text will be wrapped anyway.